### DISCUSSION TITLE
Not running properly on Windows PC

### DISCUSSION THREAD

**Discussion by leonardodasneves-eaton** [adobe/spectrum-css#2670] [Q&A] [answered]
We installed the button component by running:
yarn add -DW @spectrum-css/tokens @spectrum-css/typography @spectrum-css/page @spectrum-css/icon @spectrum-css/button

Then we got stuck running Specrtrum CSS on Windows 10 Enterprise with the following error running yarn start.

Windows specs:
Edition	Windows 10 Enterprise
Version	22H2
Installed on	‎05/‎09/‎2023
OS build	19045.4170
Experience	Windows Feature Experience Pack 1000.19054.1000.0

PS C:\Users\xxxxxxxx\repos\xxxxx-spectrum-css\spectrum-css> yarn start
yarn run v1.22.22
$ cross-env NODE_ENV=development nx start storybook

 NX   Running target start for project storybook and 5 tasks it depends on:

——————————————————————————————————————————————————————————————————————————————————————————————————————————— 

> nx run ui-icons:clean  [existing outputs match the cache, left as is]

> nx run tokens:clean

'test' is not recognized as an internal or external command,
operable program or batch file.

> nx run tokens:style-dictionary

CSS
No properties for global-vars.css. File not created.
No properties for spectrum/global-vars.css. File not created.
No properties for express/global-vars.css. File not created.
No properties for medium-vars.css. File not created.
No properties for large-vars.css. File not created.
No properties for light-vars.css. File not created.
No properties for dark-vars.css. File not created.
No properties for darkest-vars.css. File not created.
No properties for spectrum/medium-vars.css. File not created.
No properties for spectrum/large-vars.css. File not created.
No properties for spectrum/light-vars.css. File not created.
No properties for spectrum/dark-vars.css. File not created.
No properties for spectrum/darkest-vars.css. File not created.
No properties for express/medium-vars.css. File not created.
No properties for express/large-vars.css. File not created.
✅ Successfully built design tokens
✅ Successfully built design tokens

> nx run tokens:build

'cat' is not recognized as an internal or external command,
operable program or batch file.

———————————————————————————————————————————————————————————————————————————————————————————————————————————

 NX   Running target start for project storybook and 5 tasks it depends on failed

Failed tasks:

- tokens:build

Hint: run the command with --verbose for more details.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by castastrophe** [comment, marked as answer]
Hi @leonardodasneves-eaton. Thanks for reaching out.  This is a known issue and at this time, we don't support token building in Windows machines. We have work in the backlog to improve this workflow however it's low priority as our team do not use Windows machines.  Keep an eye on #2500.
